deploy/helm/sumologic/upgrade-2.0.0.sh

```diff
@@ -177,6 +177,25 @@ function migrate_prometheus_operator_to_kube_prometheus_stack() {
   yq d -i "${TEMP_FILE}" "prometheus-operator"
 }
 
+function migrate_prometheus_retention_period() {
+  if [[ -z "$(yq r "${TEMP_FILE}" -- 'prometheus-operator')" ]]; then
+    return
+  fi
+  local RETENTION
+  readonly RETENTION="$(yq r "${TEMP_FILE}" -- 'prometheus-operator.prometheus.prometheusSpec.retention')"
+
+  if [[ -z "${RETENTION}" ]]; then
+    return
+  fi
+
+  if [[ "${RETENTION}" == "7d" ]]; then
+    info "Changing prometheus retention period from 7d to 1d"
+  elif [[ "${RETENTION}" != "1d" ]]; then
+    warning "Changing prometheus retention period from (custom set) ${RETENTION} to 1d"
```

Comment:
I don't think we should change it in such case.

Reply:
I mean - if someone changed that from the default 7d they probably knew what they were doing.
I think we should rather inform user about the change of the default value for the chart and explicitly say that we leave it as it is.

Reply:
Changed, please recheck.

Reply:
Thanks :)